Update warehouse-setups-cloud-callout.md (#7414)
fishtown-analytics/docs.getdbt.com: website/snippets/warehouse-setups-cloud-callout.md

> :::info `profiles.yml` file is for dbt Core and users only

Reviewer: ```suggestion
:::info `profiles.yml` file is for local development with dbt Core and dbt fusion only
```

> :::info `profiles.yml` file is for local development with dbt Core and dbt fusion only

Reviewer 2: No a blocker but since this will sort of be a header we could shorten it
```suggestion
:::info `profiles.yml` file is for dbt Core and dbt fusion only
```

> If you're using <Constant name="dbt_platform" />, you don't need to create a `profiles.yml` file. This file is only when you use dbt Core or run dbt Fusion locally. To connect your data platform to <Constant name="cloud" />, refer to [About data platforms](/docs/cloud/connect-data-platform/about-connections). 

Reviewer: ```suggestion
If you're using <Constant name="dbt_platform" />, you don't need to create a `profiles.yml` file. This file is only necessary when you use dbt Core or dbt Fusion locally. To connect your data platform to <Constant name="cloud" />, refer to [About data platforms](/docs/cloud/connect-data-platform/about-connections). 
```
